CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ We've deprecated the `govuk-header__link--service-name` class, and will remove i
 
 This change was introduced in [pull request #2617: Do not make the service name in the header a link if no `serviceUrl` is provided](https://github.com/alphagov/govuk-frontend/pull/2617).
 
+### New features
+
+#### Help users navigate through pages with pagination
+
+You can now use [pagination](https://design-system.service.gov.uk/components/pagination/) to help users navigate forwards and backwards through a series of pages. For example, in search results or guidance that's divided ito multiple website pages.
```

Comment:
The linked guidance here doesn't exist yet so this is a broken link, but this is the page it is going to be once it gets published. There's a decision here about if we want to merge this now or once the guidance is deployed.

Reply:
We can merge this in, even with the broken link. While it's unreleased, no one should really be looking at / expect the Changelog to be fully working anyway. When we do the release, it does mean the release notes will contain a broken link for a short period of time until the guidance has been published, but we try to do them in quick succession to limit the length of time this happens. Ideally, most people will become aware of the release via our comms too, which we'll only put out once the guidance has also been published.

TLDR: it's all good to be merged, *after v4.1.1 has been released.*